simplq/package.json

```diff
@@ -36,13 +36,14 @@
     "styled-components": "^5.2.1"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
+    "start": "npm run config && react-scripts start",
+    "build": "npm run config && react-scripts build",
```

Comment:
We are configuring the url before build. Can we configure UI to take up the backend url dynamically after build?

Easiest way I could think of is to generate a json config object in the public folder and vary it from inside the frontend server. Then the client would request for this json object and derive the backend URL at start. 

I don't know if will lead to issues where a backend request happens before the backend URL is resolved.

Reply:
@daltonfury42 I don't think it should be so complicated.